docs/how-to/search.rst

```diff
@@ -0,0 +1,66 @@
+.. _search:
+
+##########################################
+Search functionality
+##########################################
+
+****************************
+Default search functionality
+****************************
+
+By default, the :ref:`article_search_plugin` plugin searches through articles' *Title* and
+*Lead-in* fields. To have it search through the main content of articles, you need to add a
+setting::
+
+    ALDRYN_NEWSBLOG_UPDATE_SEARCH_DATA_ON_SAVE = True
+
+When this is enabled, when articles and plugins are saved they will render their content, and this
```

Comment:
maybe: they will render their content and **store it in the database**. This rendered content will be added..